docs/web/docs/4-faq/index.mdx

```diff
@@ -0,0 +1,128 @@
+
+Certainly not! You are free to continue using other tools like Terraform, ClickOps, etc., for managing your cloud identities,
+while employing IAMbic to handle only a portion of them. Ideal initial candidates for IAMbic management include:
+
+1) Service Control Policies
+2) Cookie-cutter AWS Roles/Permission sensitive (Such as your `engineering` role that is used by all engineers)
+3) AWS Managed Policies
+4) Sensitive Applications or groups in Okta, Azure AD, or Google Workspace
+
+Regardless of whether you manage them via IAMbic or not, IAMbic will consistently track changes to all your identities.
+The fundamental principle here is storing all your IAM in one place, in a human-readable, bi-directional format. IAMbic gives you a
+nified view in Git instead of logging into each cloud console or using a different tool for each cloud provider.
+
+### How does IAMbic work in a production environment?
+
+IAMbic provides all essential tools and Github Actions to facilitate quick setup in a production environment. These Github actions execute the following tasks:
+
+These GitHub actions perform the following tasks:
+
+1) Regularly import IAM changes into Git, ensuring that Git remains "eventually consistent" with your permissions across AWS, Okta, Azure AD, and Google Workspace.
+2) Identify changes in near-real-time for AWS, utilizing AWS EventBridge rules monitoring IAM changes events through CloudTrail.
+3) Terminate access and permissions defined to expire within IAMbic templates.
+4) Enforce drift prevention - If a user designates IAMbic as the "source of truth" in a template, IAMbic will periodically revert any out-of-band changes to the source of truth as dictated by the IAMbic template.
+
+### What is the recommended approach for multiple users making concurrent changes to their permission templates?
+
+In a production setting, users should not directly run iambic apply. Instead, all interactions with IAMbic should be conducted through a Git flow, with IAMbic being operated by your CI/CD process.
+
+This means that users would make modifications to their templates in Git and then submit a pull request to apply these changes. This pull request would then undergo a review process by a peer or a cloud administrator before being merged.
+
+In cases where there are multiple pull requests proposing changes to the same template, these requests should be rebased on top of the most recent version of the template. For more detailed information and examples of pull requests, please refer to the
+[IAMOps Philosophy](/reference/iamops_philosophy) section.
+
+
+### What differentiates the free version of IAMbic from the paid Noq Platform?
+
+The Noq Platform provides additional features such as:
+
+1) An intuitive self-service wizard enabling end-users to request temporary or permanent access to applications, AWS credentials, or IAM permissions, without any requirement to understand IAMbic or IAM syntax.
```

Comment:
May be worth mentioning that admins can easily create their own self service requests